### DISCUSSION TITLE
最新的版本闪退是不是因为表情？

### DISCUSSION THREAD

**Discussion by fs1314** [leiurayer/downkyi#873] [Q&A]
159版本，下载的时候现在没带表情了
就是B站很多视频标题带有表情，但是现在下载下来不带表情了，（之前是可以的）。
然后下载标题带表情的视频会导致软件闪退？？

**Comment by 1tobef** [comment]
我更加可悲 🥲🥲🥲 粘贴网址,开始搜索就已经闪退>>>>>>>
1. 我目前在马来西亚, 需要翻墙回国内吗?
2. 我目前没有 +86 电话不能注册用户, 不懂`是否是这原因, 哪位大神可以借个 B 站户口尝试 🫰🫰
3. .net 已经是 4.8, WIN 11, 360 安全卫士, 尝试了 DownKyi-1.5.9 , DownKyi-1.5.8-fix1 , 都是一样 搜索闪退
